core/src/test/java/org/bitcoinj/core/BlockTest.java

```diff
@@ -38,17 +38,16 @@
 import org.junit.Test;
 
 import java.io.ByteArrayOutputStream;
-import java.io.IOException;
-import java.io.OutputStream;
 import java.math.BigInteger;
+import java.nio.Buffer;
```

Comment:
It's so nice to see `IOException` disappearing along with the change to `Buffer`.